pipeline-templates/job/code-build.yml

```diff
@@ -0,0 +1,72 @@
+jobs:
+- job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
+  variables:
+  - group: Prod Management Resources
+  - name: buildConfiguration
+    value: release
+  - group: Prod Management Resources
+  - name: buildPlatform
+    value: anycpu
+  workspace:
+    clean: all
+  steps:
+  - task: UseGitVersion@5
+    displayName: GitVersion
+    inputs:
+      versionSpec: 5.x
+      useConfigFile: true
+      configFilePath: GitVersion.yml
+
+  - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-blocks
+
+  - template: azure-pipelines-templates/build/step/dependency-check.yml@das-platform-building-blocks
+
+  - task: DotNetCoreCLI@2
+    displayName: Restore
+    inputs:
+      command: restore
+      projects: 'src/**/*.csproj'
+      noCache: true
+
+  - task: DotNetCoreCLI@2
```

Comment:
using the app-build.yml building block now so these restore/build/test etc steps are not needed